modules/security_group/main.tf

```diff
@@ -0,0 +1,31 @@
+resource "aws_security_group" "alb" {
+  name        = "${var.namespace}-alb-sg"
+  description = "ALB Security Group"
+  vpc_id      = var.vpc_id
+
+  tags = {
+    Name = "${var.namespace}-alb-sg"
+  }
+}
+
+#tfsec:ignore:aws-ec2-no-public-ingress-sgr
+resource "aws_security_group_rule" "alb_ingress_http" {
+  type              = "ingress"
+  security_group_id = aws_security_group.alb.id
+  protocol          = "tcp"
+  from_port         = 80
+  to_port           = 80
+  cidr_blocks       = ["0.0.0.0/0"]
+  description       = "From HTTP to ALB"
+}
```

Comment:
how about:
- ALB rules to redirect HTTP connection -> HTTPS
- security group rules allow HTTPS traffic only (port 443)

example: https://github.com/nimblehq/worpt-infrastructure/blob/c4e8f8f3641a095d09c683484489e5caf450c6fb/modules/alb/main.tf#L41-L55

Reply:
For IC, do we really need SSL @longnd :D, if the SSL is required, then the developer need to purchase a domain as well :D

Reply:
Yes, we cannot utilize HTTPS with the domain that ALB provided; we need to attach the domain and certificate to AWS if we wish to use HTTPS here.

https://stackoverflow.com/questions/68567684/why-does-awss-application-load-balancer-require-a-custom-domain-when-using-http

@longnd What do you think?

Reply:
yes, you are right. I did not think/forget that we don't have a domain for used in the IC, thus can't issue a certificate for use by the ALB. I will resolve this comment 🙏